### PR TITLE
add comments on code and set up linters

### PR DESCRIPTION
**Pull request by HSMathebula**
## Admission trial code review

In this code I:

- add comments in the style.css file
- setup linter and fix errors

### CODE REVIEW

**Review by HSMathebula — COMMENTED** [review]
please update the readme file

**Review by HSMathebula — COMMENTED** [review]
please make sure there are no duplicates on your css

**Comment by sja-thedude** [comment]
> please update the readme file

Kindly add this comment to your readme file as an inline comment.

**Comment by sja-thedude** [comment]
> please make sure there are no duplicates on your css

Kindly add this comment to your readme file as an inline comment.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @HSMathebula, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Good job working on the project.
✔️ Linters are passing.

## Required 
- [x] Kindly make sure to add 2 inline comments in HTML or CSS file for this project.
- [x] Kindly leave a comment summary for the project.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -1,56 +1,52 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Survey form: Admission trail project
 
-> Description the project.
+<!-- please update the readme file -->
+<!-- please make sure there are no duplicates on your css -->
```
Comment (Mre55) [inline README.md]: As requested in the [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/review_your_project.md#project-requirements) and by the previous reviewer, you're supposed to add these comments as inline comments, not to your `README.md` file. Kindly, follow [this link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-use-github-features) on how to add inline comments to your codebase and finally remove all comments from your `README.md` file to keep your file clean.

**Review by Mre55 — CHANGES_REQUESTED** [review]
**Hi @HSMathebula,**

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1,56 +1,52 @@
...
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
+👤 **Happen Mathebula**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```
Comment (HSMathebula) [inline README.md:53]: add more than one link

File: README.md [hunk 3]
```diff
@@ -1,56 +1,52 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
```
Comment (HSMathebula) [inline README.md:3]: please update the readme file

File: README.md [hunk 4]
```diff
@@ -1,56 +1,52 @@
...
+- Then go to the main folder using the next command:
 
-### Install
+$ cd Portfolio-setup-and-mobile-version-skeleton.git
 
-### Usage
+- Finally, run diplay the index.html file in your local browser, or also, you can use Live Server in Visual Studio Code.
 
-### Run tests
+### Prerequisites
```
Comment (HSMathebula) [inline README.md:31]: please remove headings you do not use

**Review by HSMathebula — COMMENTED** [review]
N/A

**Review by Mre55 — APPROVED** [review]
**Hi @HSMathebula,**

# Status: APPROVED  💪 👍 

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 

Alt Text

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**